docker-compose.yml

```diff
@@ -105,12 +105,22 @@ services:
       - POSTGRES_DATABASE_PASSWORD=${POSTGRES_DATABASE_PASSWORD}
       - POSTGRES_DATABASE_HOST=${POSTGRES_DATABASE_HOST}
       - POSTGRES_DATABASE_PORT=${POSTGRES_DATABASE_PORT}
-      - OPENAI_API_TYPE=${OPENAI_API_TYPE}
+      # R2R DB variables
```

Comment:
Why are these new variables required? Can we reuse the existing one?

Reply:
POSTGRES_VECS_COLLECTION is the required variable for fetching collection name in R2R. So its needed. The next one INDEXER_TYPE is to determine what indexing method we are to use (normal default route or r2r route). Its the same for retriever too. Rest of the env are reused as much as possible.

Reply:
I mean variables like `POSTGRES_PASSWORD`. Can we reuse `POSTGRES_DATABASE_PASSWORD`? 
Similarly for others as well.

Reply:
r2r package requires them internally. Have tried removing it and it will throw asking for the default set env values like those 5 present above. The error is attached below for ur reference.

<img width="1093" alt="Screenshot 2024-07-17 at 7 06 54 PM" src="https://github.com/user-attachments/assets/563d8cdd-3889-4368-9219-b8ef42815c8a">

Reply:
Can we set the required ENV variables through python so that we have a consistent ENV convention?

Reply:
Yes sure. Working on it